easyrsa3/easyrsa

```diff
@@ -655,6 +665,11 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
 		"$EASYRSA_OPENSSL" ecparam -in "$EASYRSA_ALGO_PARAMS" -genkey | \
 			"$EASYRSA_OPENSSL" ec -out "$out_key_tmp" $crypto_opts ${EASYRSA_PASSOUT:+-passout "$EASYRSA_PASSOUT"} || \
 			die "Failed create CA private key"
+	elif [ "ed" = "$EASYRSA_ALGO" ]; then
+                CURVE_CAPS=$(echo $EASYRSA_CURVE | tr '[a-z]' '[A-Z]')
```

Comment:
Windows does not have access to `tr`

Reply:
@TinCanTech Unfortunately I'm not able to test on Windows due to setup issues.
I wanted to know if it will be possible to perform the caps operation with `${EASYRSA_CURVE^^}` instead.

Reply:
Until the uppercase operation is figured out, I've hardcoded the checks for curves `ED25519` and `ED448` in the new commit.